libtenzir/src/connect_to_node.cpp

```diff
@@ -174,19 +174,6 @@ connect_to_node(caf::scoped_actor& self, const caf::settings& opts) {
     return caf::make_error(ec::system_error,
                            "failed to connect to node: handle is invalid");
   }
-  self->request(*result, timeout, atom::get_v, atom::version_v)
-    .receive(
-      [&](record& remote_version) {
-        // TODO
-        (void)details::check_version(remote_version);
-      },
-      [&](caf::error& error) {
-        result = caf::make_error(ec::version_error,
-                                 fmt::format("failed to receive remote version "
-                                             "within specified "
-                                             "connection-timeout of {}: {}",
-                                             timeout, std::move(error)));
-      });
```

Comment:
Where is the second version check? I could not find it in `connector.cpp`.

Reply:
It's been a few weeks since I last looked at it, but I can't find it either. Let me just close this. Not sure what I was thinking here.